src/Components/Dashboard/Feeds.js

```diff
@@ -6,25 +6,42 @@ import {
     EmojiEmotionsOutlined,
     ThumbUpAltRounded,
 } from '@material-ui/icons';
-import { getBackgroundColor } from '../../Utils/colors';
 import { ColorRing, Oval } from 'react-loader-spinner';
-import Unavailiabe from '../../Utils/unavailiable1.png';
 import InfiniteScroll from 'react-infinite-scroll-component';
 import { getMoreFeeds } from '../../Auth/Actions/feedActions';
 import { useDispatch } from 'react-redux';
-
+import {
+    GetSingleArticle,
+    LikeArticles,
+} from '../../Auth/Actions/articleActions';
+import { GetSingleGif, LikeGif } from '../../Auth/Actions/gifActions';
+import ViewGifModal from './ViewGifModal';
+import GifCommentModal from './GifCommentModal';
+import ArticleCommentModal from './ArticleCommentModal';
+import Comments from './Comments';
+import { ProfilePicture } from '../../Utils/ProfilePicture';
 const Feeds = ({
     feeds,
+    userData,
     setPostArticle,
     setPostArticleModal,
     offSet,
     setOffSet,
     feedsLength,
     requesting,
     feedsTotal,
+    clickedImage,
+    setClickedImage,
+    open,
+    setOpen,
 }) => {
+    console.log(feeds);
```

Comment:
console

Reply:
It was a mistake sir, but i have remove it now